lib/libzfs/os/linux/libzfs_sendrecv_os.c

```diff
@@ -24,29 +24,9 @@
 
 #include "libzfs_impl.h"
 
-#ifndef F_SETPIPE_SZ
-#define	F_SETPIPE_SZ (F_SETLEASE + 7)
-#endif /* F_SETPIPE_SZ */
-
-#ifndef F_GETPIPE_SZ
-#define	F_GETPIPE_SZ (F_GETLEASE + 7)
-#endif /* F_GETPIPE_SZ */
 
 void
 libzfs_set_pipe_max(int infd)
```

Comment:
any reason to keep this unused file?

Reply:
Nope. I thought I had git rm'd it along with the Makefile update.